chirp/wxui/clone.py

```diff
@@ -101,7 +102,7 @@ def _add_grid(label, control):
         _add_grid('Vendor', self._vendor)
         self.Bind(wx.EVT_CHOICE, self._selected_vendor, self._vendor)
 
-        self._model = wx.Choice(panel, choices=[], style = wx.CB_SORT)
+        self._model = wx.Choice(panel, choices=[])
```

Comment:
Why are you reverting this now?

Reply:
This is my mistake! Sorting the models in th wx.Choice causes return of wrong index when trying to load the radio class!

Reply:
Ah, okay. Probably better to revert that as a single commit with an explanation, but okay.